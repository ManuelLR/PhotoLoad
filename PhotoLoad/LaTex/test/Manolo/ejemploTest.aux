\relax 
\providecommand\hyper@newdestlabel[2]{}
\catcode `"\active 
\catcode `<\active 
\catcode `>\active 
\@nameuse{es@quoting}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\select@language{spanish}
\@writefile{toc}{\select@language{spanish}}
\@writefile{lof}{\select@language{spanish}}
\@writefile{lot}{\select@language{spanish}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.1}Prueba FL1}{1}{subsection.0.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.2}Prueba FL2}{1}{subsection.0.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.3}Prueba FL3}{1}{subsection.0.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.4}Prueba FB1}{1}{subsection.0.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.5}Prueba FB2}{2}{subsection.0.5}}
