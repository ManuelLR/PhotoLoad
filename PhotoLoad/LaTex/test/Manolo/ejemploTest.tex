% !TeX spellcheck = es_ES
\documentclass{scrartcl}
%\documentclass{book}
\usepackage[spanish]{babel}
\usepackage[utf8]{inputenc}
\usepackage{enumerate}
\usepackage{graphicx}
\usepackage{color}
\usepackage{float}
\usepackage{multicol}
\usepackage{lscape}
\usepackage{fancyhdr}
\usepackage{tabularx}
\usepackage[hidelinks,colorlinks=true, linkcolor=black]{hyperref} %hyperref para hacer los links del indice y usar \url{URL} y \href{URL}{text}y hidelinks para que no se rodeen con una caja de color los enlaces. Más información en: http://en.wikibooks.org/wiki/LaTeX/Hyperlinks
\pagestyle{fancy}
\begin{document}


\subsection{Prueba FL1}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & Prueba FL1 \\ 
	\hline \textbf{Descripción} & Comprueba el correcto funcionamiento de las dos fases de login necesarios para Flickr \\	 
	\hline  \textbf{Entrada}		& No entra ningún dato \\ 
	\hline  \textbf{Salida esperada}			& Un objeto de tipo OAuth con una URL y un AccessToken relleno\\
	\hline  \textbf{Resultado}			& Exito \\
	\hline 
\end{tabularx} 

\subsection{Prueba FL2}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & Prueba FL2 \\ 
	\hline \textbf{Descripción} & Comprueba la correcta obtención de las fotos existentes en Flickr para el usuario \\	 
	\hline  \textbf{Entrada}		& Token (OAuth) \\ 
	\hline  \textbf{Salida esperada}			& Lista de FlickrPhoto no null con los ids de las fotos pero sin los links de descarga\\
	\hline  \textbf{Resultado}			& Exito \\
	\hline 
\end{tabularx} 

\subsection{Prueba FL3}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & Prueba FL3\\ 
	\hline \textbf{Descripción} & Comprueba la correcta obtención de los links de las fotos existentes en Flickr para el usuario \\	 
	\hline  \textbf{Entrada}		& Token (OAuth) y lista de FlickrPhoto de las que queremos obtener el link \\ 
	\hline  \textbf{Salida esperada}			& Lista de FlickrPhoto con la información anterior además de links de descarga\\
	\hline  \textbf{Resultado}			& Exito \\
	\hline 
\end{tabularx} 

\subsection{Prueba FB1}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & Prueba FB1\\ 
	\hline \textbf{Descripción} & Comprueba la correcta obtención de las fotos existentes en Facebook para el usuario \\	 
	\hline  \textbf{Entrada}		& Token \\ 
	\hline  \textbf{Salida esperada}			& Lista de Fotos\\
	\hline  \textbf{Resultado}			& Exito \\
	\hline 
\end{tabularx} 

\subsection{Prueba FB2}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & Prueba FB2\\ 
	\hline \textbf{Descripción} & Comprueba la correcta subida de fotos al tablon de Facebook \\	 
	\hline  \textbf{Entrada}		& Token y url de la foto a subir \\ 
	\hline  \textbf{Salida esperada}			& El id de la foto subida\\
	\hline  \textbf{Resultado}			& Exito \\
	\hline 
\end{tabularx} 
\end{document}