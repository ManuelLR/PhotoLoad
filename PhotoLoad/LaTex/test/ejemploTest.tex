% !TeX spellcheck = es_ES
\documentclass{scrartcl}
%\documentclass{book}
\usepackage[spanish]{babel}
\usepackage[utf8]{inputenc}
\usepackage{enumerate}
\usepackage{graphicx}
\usepackage{color}
\usepackage{float}
\usepackage{multicol}
\usepackage{lscape}
\usepackage{fancyhdr}
\usepackage{tabularx}
\usepackage[hidelinks,colorlinks=true, linkcolor=black]{hyperref} %hyperref para hacer los links del indice y usar \url{URL} y \href{URL}{text}y hidelinks para que no se rodeen con una caja de color los enlaces. Más información en: http://en.wikibooks.org/wiki/LaTeX/Hyperlinks
\pagestyle{fancy}
\begin{document}


\subsection{prueba 1}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & prueba1 \\ 
	\hline \textbf{Descripción} & Esta es una descripcion muy muy muy larga que se ajustará automáticamente a la celda
	
	y esto es lo que pasa si quiero metertle más espacio \\	 
	\hline  \textbf{Entrada}		& A escribir \\ 
	\hline  \textbf{Salida esperada}			& Po JIIII \\
	\hline  \textbf{Resultado}			& Fracaso \\
	\hline 
\end{tabularx} 

\subsection{prueba 2}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & prueba2 \\ 
	\hline \textbf{Descripción} & Esta es una descripcion muy muy muy larga que se ajustará automáticamente a la celda
	
	y esto es lo que pasa si quiero metertle más espacio \\	 
	\hline  \textbf{Entrada}		& A escribir \\ 
	\hline  \textbf{Salida esperada}			& Po JIIII \\
	\hline  \textbf{Resultado}			& Fracaso \\
	\hline 
\end{tabularx} 
\end{document}