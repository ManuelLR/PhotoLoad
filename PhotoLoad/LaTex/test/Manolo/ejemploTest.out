\BOOKMARK [2][-]{subsection.0.1}{Prueba FL1}{}% 1
\BOOKMARK [2][-]{subsection.0.2}{Prueba FL2}{}% 2
\BOOKMARK [2][-]{subsection.0.3}{Prueba FL3}{}% 3
\BOOKMARK [2][-]{subsection.0.4}{Prueba FB1}{}% 4
\BOOKMARK [2][-]{subsection.0.5}{Prueba FB2}{}% 5
